modules/configuring-ide.adoc

```diff
@@ -0,0 +1,199 @@
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a Code Server workbench.
+* You have installed Docker on your system.
+
+.Procedure
+
+. Clone the Open Data Hub *notebooks* repository:
++
+----
+git clone git@github.com:opendatahub-io/notebooks.git
+----
+
+. From your local version of the repository, go to the `codeserver` folder and create a new folder for your custom image. For example, `ubi9-python-3.9-db-clients`. 
+. Inside this folder, create a Dockerfile with the following contents:
++
+----
+# The base image is auto-assigned by the make recipe on the next step. In this case, this is your Code Server notebook.
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+WORKDIR /opt/app-root/bin
+
+# Install OS packages as root
+USER root
+
+# Install necessary OS packages
+RUN dnf install -y unixODBC postgresql
+
+# Install MongoDB Client
+COPY mongodb-org-6.0.repo-x86_64 /etc/yum.repos.d/mongodb-org-6.0.repo
+RUN dnf install -y mongocli
+
+# Install MSSQL Client
+COPY mssql-2022.repo-x86_64 /etc/yum.repos.d/mssql-2022.repo
+RUN ACCEPT_EULA=Y dnf install -y mssql-tools18 unixODBC-devel
+ENV PATH="$PATH:/opt/mssql-tools18/bin"
+
+# Switch back to default user
+USER 1001
+
+WORKDIR /opt/app-root/src
+----
+
+. Create two RPM files, `mongodb-org-6.0.repo-x86_64` and `mssql-2022.repo-x86_64`, with the following contents, in the folder you created for your custom image.
++
+*`mongodb-org-6.0.repo-x86_64`:*
+
++
+----
+[mongodb-org-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/9/mongodb-org/6.0/x86_64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+----
+
++
+*`mssql-2022.repo-x86_64`:*
+
++
+----
+[packages-microsoft-com-prod]
+name=packages-microsoft-com-prod
+baseurl=https://packages.microsoft.com/rhel/9.0/prod/
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.microsoft.com/keys/microsoft.asc
+----
+
+. To streamline the build and push process, update your local version of the link:https://github.com/opendatahub-io/notebooks/blob/main/Makefile[makefile] with a new recipe:
++
+----
+.PHONY: codeserver-ubi9-python-3.9-db-clients
+codeserver-ubi9-python-3.9-db-clients: codeserver-ubi9-python-3.9
+    $(call image,$@,codeserver/ubi9-python-3.9-db-clients,$<)
+----
+
+. Run the following command to build and push the image:
++
+Replace `${YOUR_USERNAME}` with your username. You can replace `quay.io` with any valid registry.
+
++
+----
+make codeserver-ubi9-python-3.9-db-clients -e IMAGE_REGISTRY=quay.io/${YOUR_USERNAME}/workbench-images
+----
+
+. After pushing the custom image, go to *Settings* -> *Notebooks Image Settings* -> *Import New Image*.
```

Comment:
accessed through the dashboard.

Reply:
Updating to:
_After pushing the custom image, from the {productname-short} dashboard, go to *Settings* -> *Notebooks Image Settings* -> *Import New Image*._

Reply:
that's nice, thanks.